src/contexts/WizardFormContext.tsx

```diff
@@ -3,6 +3,8 @@ import FormScreens from 'constants/formScreens';
 import StepOrder from 'constants/stepOrder';
 import { FormValuesType, ParentScreens, Screens } from 'types/formTypes';
 
+export const initialFirstUncompletedStep = FormScreens.PERSONAL_DETAILS;
```

Comment:
You don't need to put it as a separate const. Just  pass enum value to `initialValue` of context